src/static/js/modules/init-chosen.js

```diff
@@ -10,7 +10,9 @@ require( 'chosen' );
 function init() {
   $( '.chosen-select' ).chosen( {
     width:           '100%',
+    /* eslint-disable */
```

Comment:
Ha, so I did that, but then it triggered three new warnings (including line length), so I gave up and went global.